src/bp-templates/bp-nouveau/js/buddypress-messages.js

```diff
@@ -2710,6 +2715,11 @@ window.bp = window.bp || {};
 					errors.push( 'message_content' );
 				}
 
+				content = content.replace(/&nbsp;/g,'').trim();
+				if($(content).text().trim() === '') {
+					errors.push( 'message_content' );
+				}
+
```

Comment:
@buddy4kp here too, this condition should not check if condition written above is met. Please apply proper logic here.